VimeoNetworking/Sources/Models/VIMUser.m

```diff
@@ -153,6 +153,15 @@ - (void)parseConnections
             if([value isKindOfClass:[NSDictionary class]])
             {
                 VIMConnection *connection = [[VIMConnection alloc] initWithKeyValueDictionary:value];
+                
+                // Parse Notification Connection keys
+                if ([key isEqualToString:VIMConnectionNameNotifications])
+                {
+                    [connection setNotifications:value];
+                }
+                
+                if([connection respondsToSelector:@selector(didFinishMapping)])
+                    [connection didFinishMapping];
```

Comment:
I don't think we should ever be calling `didFinishMapping` manually. Can we handle this connection stuff exactly like we are for the VOD stuff for consistency?